UnityProject/Assets/Scripts/HealthV2/Living/LivingHealthMasterBase.cs

```diff
@@ -316,6 +319,8 @@ public BleedingState CalculateBleedingState()
 			MultiInterestBool.RegisterBehaviour.RegisterFalse,
 			MultiInterestBool.BoolBehaviour.ReturnOnFalse);
 
+		[SerializeField, Range(1,60f)] private float updateTime = 1.25f;
```

Comment:
why did you change the time?
and also there's systems that calculate presuming each update is one sec

Reply:
this also would completely change the rate of how much you would take space damage and stuff, plus fire damage and all that stuff

Reply:
I didn't mean to change it, was just trying to move away the magic number into a variable.